src/layout/Navbar/Navbar.js

```diff
@@ -26,54 +26,76 @@ import { RoundedIcon } from 'igz-controls/components'
 
 import { getLinks } from './Navbar.utils'
 import localStorageService from '../../utils/localStorageService'
-import { NAVBAR_WIDTH } from '../../constants'
+import { NAVBAR_WIDTH_CLOSED, NAVBAR_WIDTH_OPEN } from '../../constants'
 
 import { ReactComponent as PinIcon } from 'igz-controls/images/pin-icon.svg'
 import { ReactComponent as UnPinIcon } from 'igz-controls/images/unpin-icon.svg'
 import { ReactComponent as SettingsIcon } from 'igz-controls/images/navbar/mlrun-project-settings.svg'
 
 import './Navbar.scss'
 
-const Navbar = ({ isHeaderShown, isNavbarPinned, projectName, setIsNavbarPinned }) => {
+const Navbar = ({ projectName, setIsNavbarPinned }) => {
+  const [navabrState, setNavbarState] = useState({
```

Comment:
it's better to separate states to isHovered and isPinned, in this case we won't need to use rest operator everytime we set a state